contracts/tec/contracts/src/MixinSignatureValidator.sol

```diff
@@ -0,0 +1,107 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.3;
+
+import "@0x/contracts-utils/contracts/src/LibBytes.sol";
+import "./mixins/MSignatureValidator.sol";
+
+
+contract MixinSignatureValidator is
+    MSignatureValidator
+{
+    using LibBytes for bytes;
+
+    /// @dev Recovers the address of a signer given a hash and signature.
+    /// @param hash Any 32 byte hash.
+    /// @param signature Proof that the hash has been signed by signer.
+    function getSignerAddress(bytes32 hash, bytes memory signature)
```

Comment:
Is it possible to share the `getSignerAddress` used by the Exchange? Other extensions could also make use of it as a generic lib.

Reply:
The one on the Exchange contract returns a `bool` rather than an `address` unfortunately.

Reply:
Ah okay, maybe for v3 we could refactor this. I'll make a note of it.